staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -126,7 +128,15 @@ func WithPriorityAndFairness(
 
 			handler.ServeHTTP(w, innerReq)
 		}
-		digest := utilflowcontrol.RequestDigest{RequestInfo: requestInfo, User: user}
+
+		// find the estimated "width" of the request
+		width, ok, err := widthFinder.FindWidth(r)
+		if err != nil {
+			handleError(w, r, fmt.Errorf("failed to estimate width of the request - %v", err))
+			return
```

Comment:
I'm not convinced this is a good option. Us not being able to estimate the width shouldn't fail the request.

I can see two options:
1) hide error handling in the interface (i.e. the interface always returns a number, does something clever inside)
2) handle error here in a way that:
- we expose some metric that we can track to ensure that this is not happening
- if we can't compute the width, default to something (e.g. the highest possible value)

I think that hiding that logic in the interface itself (i.e. saying that it has to handle the errors) is better (and doing what I describe in (2) internally there).